nntrainer/include/neuralnet.h

```diff
@@ -253,6 +258,8 @@ class NeuralNetwork {
    */
   TensorDim getOutputDimension() { return layers.back()->getOutputDimension(); }
 
+  FlatGraphType getFlatGraph() { return layers; }
```

Comment:
add Doxygen comments.
Also, add that this is not a deep copy of the graph (getWeights() and modifying will affect the original model layers).

Reply:
Done